addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -323,6 +323,9 @@ $popover-border-color: $border-color !default;
 $popover-arrow-color: $body-bg !default;
 $popover-arrow-outer-color: $border-color !default;
 
+// Badge
+$badge-border-radius: $border-radius-pill !default;
```

Comment:
This line is there because we want to force all the badges to be `rounded-pill` by default, indeed. 

> So now, it's no longer possible to have a badge that isn't rounded in a template?

Should still be possible by using `rounded`, if I'm not misunderstanding you ?

Reply:
Would make sense indeed